lib/api/core/plugins/pluginContext.js

```diff
@@ -200,22 +203,24 @@ class PluginContext {
         collection,
         ObjectConstructor = null
       ) {
-        let pluginRepository;
-
         if (!collection) {
           errorsManager.throw(
             'plugins',
             'validation',
             'collection_not_specified');
         }
-
-        pluginRepository =
-          new PluginRepository(kuzzle, pluginInternalIndex, collection);
-        pluginRepository.init(
-          {databaseEngine: pluginInternalEngine,
-            cacheEngine: null,
-            ObjectConstructor}
-        );
+        // @todo pass the good index name
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)